ui/App/ProjectScreen/Source/Commit.svelte

```diff
@@ -72,6 +79,38 @@
     border-top: 1px solid var(--color-foreground-level-3);
     margin: 1rem 0 1.5rem 0;
   }
+
+  .anchor {
+    background-color: var(--color-foreground-level-1);
+    border-radius: 0.5rem;
+    border: 1px solid var(--color-foreground-level-2);
+    margin-top: 1rem;
+    margin-bottom: 1rem;
+  }
+
+  .header {
+    border-bottom: 1px solid var(--color-foreground-level-2);
+    min-height: 3.5rem;
```

Comment:
We could replace this with `padding: 1rem`. This would make it consistent with how padding is used for `metadata`.

Reply:
aa9bc1f7422719b8afef06ffe2faeb48cffa0f75